Client/Frontend/Fakespot/Views/FakespotAdjustRatingView.swift

```diff
@@ -119,7 +111,6 @@ class FakespotAdjustRatingView: UIView, Notifiable, ThemeApplicable {
 
     private func adjustLayout() {
         let contentSizeCategory = UIApplication.shared.preferredContentSizeCategory
-        starRatingHeightConstraint?.constant = min(UIFontMetrics.default.scaledValue(for: UX.starSize), UX.starMaxSize)
 
```

Comment:
Removing the height constraint causes the stars to not grow with accessibility sizes. You can reproduce by increasing the font from Xcode or by adding Text Size on the control center on the iPhone.


| | |
|--------|--------|
| ![image](https://github.com/mozilla-mobile/firefox-ios/assets/26011662/2bf80b3f-5e2e-4933-8558-f19fb3bb4d04) | ![image](https://github.com/mozilla-mobile/firefox-ios/assets/26011662/5fc0f98c-f1f6-46f8-b926-c28d356a1935) |

Reply:
Thanks @razvanlitianu for pointing this out. Have updated the PR, please have a look.